requirements.txt

```diff
@@ -0,0 +1,6 @@
+jupyter==1.0.0
```

Comment:
I would rather see these with ranges, because our JL environment should have several of these packages by default, but I am going to be tracking upstream until quite close to our freeze-for-commissioning.

So maybe `>=1.0.0,<2.0.0` ?  (assuming all of these use semantic versioning)